.github/workflows/dispatch-bump-sentry.yml

```diff
@@ -0,0 +1,28 @@
+name: dispatch getsentry bump-sentry
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  dispatch:
+    name: dispatch
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Dispatch getsentry tests
+        uses: actions/github-script@v6
```

Comment:
Should we pin this to a sha?

Reply:
deferring this to when i have some time to get to https://getsentry.atlassian.net/jira/software/c/projects/DEVINFRA/boards/172?modal=detail&selectedIssue=DEVINFRA-38 where i'll just pin everyhting